hashtagsv2/hashtags/views.py

```diff
@@ -53,12 +53,16 @@ def get_queryset(self):
         form = self.form_class(self.request.GET)
         if form.is_valid():
             form_data = form.cleaned_data
-
-            hashtag_qs = hashtag_queryset(form_data)
-
-            if hashtag_qs.count() == 0:
+            if(form_data['project'] == 'www.wikidata.org'):
+                hashtag_qs = []
                 messages.add_message(self.request, messages.INFO,
-                    'No results found.')
+                    'Currently www.wikidata.org is not supported as the project.')
```

Comment:
Could we reword this slightly, to "Unfortunately Wikidata searching is not currently supported."? :)